src/test/java/VehicleTest.java

```diff
@@ -251,87 +218,81 @@ public void testDrive8() {
 	
 	@Test
 	public void testMPG0() {
-		assertTrue(3 == testBoat1.getAdjustedMPG(4));
-		
-	}
-	
-	@Test
-	public void testMPG1() {
-		assertTrue(3 == testBoat2.getAdjustedMPG(4));
+		assertEquals(3, testBoat1.getAdjustedMPG(4));
 		
 	}
 	
 	@Test
 	public void testMPG2() {
-		assertTrue(5 == testBoat3.getAdjustedMPG(1));
+		assertEquals(5, testBoat3.getAdjustedMPG(1));
 		
 	}
 	@Test
 	public void testMPG3() {
-		assertTrue(3 == testBoat3.getAdjustedMPG(2));
+	    assertEquals(3, testBoat3.getAdjustedMPG(2));
 		
 	}
 	
 	@Test
 	public void testMPG4() {
-		assertTrue(1 == testBoat4.getAdjustedMPG(4));
+	    assertEquals(1, testBoat4.getAdjustedMPG(4));
 		
 	}
 	
 	@Test
 	public void testMPG5() {
-		assertTrue(3 == testBoat4.getAdjustedMPG(3));
+	    assertEquals(3, testBoat4.getAdjustedMPG(3));
 		
 	}
 	
 	@Test
 	public void testFuel1() {
 		testCar.drive(15, 3);
-		assertTrue(5 == testCar.fuelGauge());
+		assertEquals(5, testCar.fuelGauge());
 	}
 	
 	@Test
 	public void testFuel2() {
 		testCar.drive(30, 3);
-		assertTrue(0 == testCar.fuelGauge());
+		assertEquals(0, testCar.fuelGauge());
```

Comment:
almost never use assertTrue - it's nearly never the correct thing (unless it's a boolean you're testing)